daml/Marketplace/Issuer.daml

```diff
@@ -55,7 +55,7 @@ template Issuer
           observers : [Party]
         do
           let tokenId = Id with signatories = fromList [issuer], label = name, version = 0
-          let tokenObservers = if isPublic then (public :: observers) else observers
+          let tokenObservers = if isPublic then (public :: operator :: observers) else (operator :: observers)
```

Comment:
Why are we inserting the operator here? Previously we had taken the operator out of a lot of these contracts for privacy

Reply:
Inserting the operator here when `isPublic` is necessary since I had to change to use the operator trigger instead of an issuer trigger (since the original issuer automation used the public party that the trigger doesn't have access to). 

So in the case where we were already adding `Public`, it's no less private...however when it isn't `isPublic`, I shouldn't be adding the operator - good catch, I will fix that.

Reply:
Sounds good!